src/renderer/corporate/member/corporate-member-container.jsx

```diff
@@ -28,13 +41,29 @@ const individualFields = [
 	'phoneNumber'
 ];
 
-class CorporateAddMemberContainer extends PureComponent {
+class CorporateMemberContainerComponent extends PureComponent {
 	constructor(props) {
 		super(props);
+		const { member, profile } = props;
+
 		this.state = {
 			errors: { hasErrors: false },
-			type: 'individual',
-			parentId: props.companies[0].identity.id
+			type: profile ? profile.identity.type : 'individual',
+			equity: profile ? profile.identity.equity : '',
+			positions: new Set(profile ? profile.identity.positions : []),
```

Comment:
when position is updated set will not change referrence, so component will not be rerendered

Reply:
Fixed on 6913f4e